doc/source/reference/routines.polynomials.rst

```diff
@@ -9,24 +9,153 @@ of the `numpy.polynomial` package, introduced in NumPy 1.4.
+code.
+Since NumPy version 1.4, the `numpy.polynomial` package is preferred for
+working with polynomials.
+
+Quick Reference
+~~~~~~~~~~~~~~~
+
+The following table highlights some of the main differences between the
+legacy polynomial module and the polynomial package for common tasks.
+The `~numpy.polynomial.polynomial.Polynomial` class is imported for brevity::
+
+    from numpy.polynomial import Polynomial
+
+
++------------------------+------------------------------+---------------------------------------+
+|  **How to...**         | Legacy (`numpy.poly1d`)      | `numpy.polynomial`                    |
++------------------------+------------------------------+---------------------------------------+
+| Create a               | ``p = np.poly1d([1, 2, 3])`` | ``p = Polynomial([3, 2, 1])``         |
+| polynomial object      |                              |                                       |
+| from coefficients [1]_ |                              |                                       |
++------------------------+------------------------------+---------------------------------------+
+| Create a polynomial    | ``r = np.poly([-1, 1])``     | ``p = Polynomial.fromroots([-1, 1])`` |
+| object from roots      | ``p = np.poly1d(r)``         |                                       |
++------------------------+------------------------------+---------------------------------------+
+| Fit a polynomial of    |                              |                                       |
+| degree ``deg`` to data | ``np.polyfit(x, y, deg)``    | ``Polynomial.fit(x, y, deg)``         |
++------------------------+------------------------------+---------------------------------------+
+
+
+.. [1] Note the reversed ordering of the coefficients
+
+Transition Guide
+~~~~~~~~~~~~~~~~
+
+There are significant differences between ``numpy.lib.polynomial`` and
+`numpy.polynomial`.
+The most significant difference is the ordering of the coefficients for the
+polynomial expressions.
+The  various routines in `numpy.polynomial` all
+deal with series whose coefficients go from degree zero upward,
+which is the *reverse order* of the poly1d convention.
+The easy way to remember this is that indices
+correspond to degree, i.e., ``coef[i]`` is the coefficient of the term of
+degree *i*.
+
+Though the difference in convention may be confusing, it is straightforward to
+convert from the legacy polynomial API to the new.
+For example, the following demonstrates how you would convert a `numpy.poly1d`
+instance representing the expression :math:`x^{2} + 2x + 3` to a
+`~numpy.polynomial.polynomial.Polynomial` instance representing the same
+expression::
+
+    >>> p1d = np.poly1d([1, 2, 3])
+    >>> p = np.polynomial.Polynomial(p1d.coef[::-1])
+
+In addition to the ``coef`` attribute, polynomials from the polynomial
+package also have ``domain`` and ``window`` attributes.
+These attributes are most relevant when
```

Comment:
Possible typo here - "most relevant when fitting polynomials to data"?

Reply:
Indeed - nice catch!